### PR TITLE
[Issue 89][Android] Contact information about "personvernombud" in consent 1 + adjusted "personvernerklæring"-button

### PR DESCRIPTION
**Pull request by MIKKH**
Divided section below FHI contact info. into 2 parts, and added new section in-between about "personvernombud" contact info.

Replaced button to "personvernerkæring" ("Privacy Policy") with a text link.

### CODE REVIEW

File: NDB.Covid19/NDB.Covid19.iOS/NDB.Covid19.iOS.csproj [hunk 1]
```diff
@@ -538,20 +538,48 @@
...
+    </ImageAsset>
+    <ImageAsset Include="Assets.xcassets\checkboxFilled.imageset\combinedShape.png">
+      <Visible>false</Visible>
+    </ImageAsset>
+    <ImageAsset Include="Assets.xcassets\checkboxFilled.imageset\combinedShape%402x.png">
+      <Visible>false</Visible>
+    </ImageAsset>
+    <ImageAsset Include="Assets.xcassets\checkboxFilled.imageset\combinedShape%403x.png">
+      <Visible>false</Visible>
+    </ImageAsset>
```
Comment (jonasofrich) [inline NDB.Covid19/NDB.Covid19.iOS/NDB.Covid19.iOS.csproj:582]: Why are all these changed?

File: NDB.Covid19/NDB.Covid19/ViewModels/ConsentViewModel.cs [hunk 2]
```diff
@@ -22,11 +22,13 @@ public class ConsentViewModel
...
-        public static string CONSENT_FIVE_PARAGRAPH => "CONSENT_FIVE_PARAGRAPH".Translate();
+        public static string CONSENT_FIVE_PARAGRAPH_SECTION_ONE => "CONSENT_FIVE_PARAGRAPH_SECTION_ONE".Translate();
+        public static string CONSENT_FIVE_PARAGRAPH_SECTION_TWO => "CONSENT_FIVE_PARAGRAPH_SECTION_TWO".Translate();
+        public static string CONSENT_FIVE_PARAGRAPH_SECTION_THREE => "CONSENT_FIVE_PARAGRAPH_SECTION_THREE".Translate();
         public static string CONSENT_SIX_TITLE => "CONSENT_SIX_TITLE".Translate();
         public static string CONSENT_SIX_PARAGRAPH => "CONSENT_SIX_PARAGRAPH".Translate();
-        public static string CONSENT_SEVEN_BUTTON_TEXT => "CONSENT_SEVEN_BUTTON_TEXT".Translate();
-        public static string CONSENT_SEVEN_BUTTON_URL => "CONSENT_SEVEN_BUTTON_URL".Translate();
+        public static string CONSENT_SEVEN_LINK_TEXT => "CONSENT_SEVEN_BUTTON_TEXT".Translate();
+        public static string CONSENT_SEVEN_LINK_URL => "CONSENT_SEVEN_BUTTON_URL".Translate();
```
Comment (jonasofrich) [inline NDB.Covid19/NDB.Covid19/ViewModels/ConsentViewModel.cs:31]: We should consider renaming the locales key as well to avoid confusion

**Review by jonasofrich — CHANGES_REQUESTED** [review]

**Comment by jonasofrich** [comment]
Be aware of the merge conflict here as well

File: NDB.Covid19/NDB.Covid19.iOS/NDB.Covid19.iOS.csproj (hunk 1 above)
Comment (MIKKH) [reply NDB.Covid19/NDB.Covid19.iOS/NDB.Covid19.iOS.csproj:582]: I haven't changed anything in the iOS project, so I guess it must have been VS changing the files during a build operation.
I will checkout the version from the 'dev-r2_x' branch, and add to a new commit.

File: NDB.Covid19/NDB.Covid19.Droid/Resources/layout/consent_info.xml [hunk 3]
```diff
@@ -82,21 +82,39 @@
         android:layout_marginBottom="16dp" />
 
     <TextView
-        android:text="Behandling af personopl., section"
+        android:text="Behandling af personopl., section1"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         style="@style/PrimaryTextRegular"
-        android:id="@+id/consent1_behandlingafpersonopl_text"
+        android:id="@+id/consent1_behandlingafpersonopl_text_section1"
```
[the remark this answers is not in the document]
Comment (MIKKH) [reply NDB.Covid19/NDB.Covid19.Droid/Resources/layout/consent_info.xml]: I have made a new commit with English ID names.

File: NDB.Covid19/NDB.Covid19.Droid/Resources/layout/consent_info.xml [hunk 4]
```diff
@@ -82,21 +82,39 @@
...
         android:layout_below="@+id/consent_paragraph_hvordan_accepterer"
         android:layout_marginBottom="16dp" />
 
-    <Button
+    <TextView
+        android:text="Behandling af personopl., section2"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        style="@style/PrimaryTextRegular"
+        android:id="@+id/consent1_behandlingafpersonopl_text_section2"
```
[the remark this answers is not in the document]
Comment (MIKKH) [reply NDB.Covid19/NDB.Covid19.Droid/Resources/layout/consent_info.xml]: I have made a new commit with English ID names.

File: NDB.Covid19/NDB.Covid19.Droid/Resources/layout/consent_info.xml [hunk 5]
```diff
@@ -82,21 +82,39 @@
...
+        android:layout_below="@+id/consent1_behandlingafpersonopl_text_section1"
+        android:layout_marginBottom="16dp" />
+
+    <TextView
+        android:text="Behandling af personopl., section3"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        style="@style/PrimaryTextRegular"
+        android:id="@+id/consent1_behandlingafpersonopl_text_section3"
```
[the remark this answers is not in the document]
Comment (MIKKH) [reply NDB.Covid19/NDB.Covid19.Droid/Resources/layout/consent_info.xml]: I have made a new commit with English ID names.

File: NDB.Covid19/NDB.Covid19.Droid/Views/Settings/ConsentSettingPageFragment.cs [hunk 6]
```diff
@@ -21,7 +21,10 @@ public class ConsentSettingPageFragment : Fragment
         TextView _howitworksText2;
         TextView _samtykkeText1;
         TextView _samtykkeText2;
-        TextView _behandlingafpersonoplysningerText;
+        TextView _behandlingafpersonoplysningerText1;
+        TextView _behandlingafpersonoplysningerText2;
+        TextView _behandlingafpersonoplysningerText3;
```
[the remark this answers is not in the document]
Comment (MIKKH) [reply NDB.Covid19/NDB.Covid19.Droid/Views/Settings/ConsentSettingPageFragment.cs]: I have made a new commit with English variable names.

File: NDB.Covid19/NDB.Covid19.Droid/Views/Welcome/WelcomePageConsentsActivity.cs [hunk 7]
```diff
@@ -31,7 +31,10 @@ public class WelcomePageConsentsActivity: AppCompatActivity
         TextView _howitworksText2;
         TextView _samtykkeText1;
         TextView _samtykkeText2;
-        TextView _behandlingafpersonoplysningerText;
+        TextView _behandlingafpersonoplysningerText1;
+        TextView _behandlingafpersonoplysningerText2;
+        TextView _behandlingafpersonoplysningerText3;
```
[the remark this answers is not in the document]
Comment (MIKKH) [reply NDB.Covid19/NDB.Covid19.Droid/Views/Welcome/WelcomePageConsentsActivity.cs]: I have made a new commit with English variable names.

File: NDB.Covid19/NDB.Covid19.Droid/Views/Welcome/WelcomePageConsentsActivity.cs [hunk 8]
```diff
@@ -101,9 +114,9 @@ protected override void OnCreate(Bundle savedInstanceState)
             RelativeLayout RelativeLayout4 = FindViewById<RelativeLayout>(Resource.Id.consent_paragraph_hvordan_accepterer);
             RelativeLayout4.FindViewById<TextView>(Resource.Id.consent_page_text).TextFormatted = HtmlCompat.FromHtml(ConsentViewModel.CONSENT_FOUR_PARAGRAPH, HtmlCompat.FromHtmlModeLegacy);
 
-            Button policyLinkBtn = FindViewById<Button>(Resource.Id.consent_paragraph_policy_btn);
-            policyLinkBtn.Text = ConsentViewModel.CONSENT_SEVEN_BUTTON_TEXT;
-            policyLinkBtn.Click += PolicyLinkBtn_Click;
+            //Button policyLinkBtn = FindViewById<Button>(Resource.Id.consent_paragraph_policy_btn);
+            //policyLinkBtn.Text = ConsentViewModel.CONSENT_SEVEN_BUTTON_TEXT;
+            //policyLinkBtn.Click += PolicyLinkBtn_Click;
```
[the remark this answers is not in the document]
Comment (MIKKH) [reply NDB.Covid19/NDB.Covid19.Droid/Views/Welcome/WelcomePageConsentsActivity.cs]: Removed in new commit.

File: NDB.Covid19/NDB.Covid19.iOS/NDB.Covid19.iOS.csproj (hunk 1 above)
Comment (MIKKH) [reply NDB.Covid19/NDB.Covid19.iOS/NDB.Covid19.iOS.csproj]: I have made new commit without changes to the iOS project files.

**Review by jonasofrich — APPROVED** [review]
Great work!

Please remember to change the key names in the text excel sheet

**Review by Frederik-S-N-dk — APPROVED** [review]

**Comment by MIKKH** [comment]
**Screenshots of implemented changes.**

**Welcome page (Bokmål):**
Welcomepage_consent_bokmål

**Settings page (Bokmål):**
Settingspage_consent_Bokmål


**Welcome page (Nynorsk):**
Welcomepage_consent_Nynorsk

**Settings page (Nynorsk):**
Settingspage_consent_Nynorsk


**Welcome page (English):**
Welcomepage_consent_English

**Settings page (English):**
Settingspage_consent_English

**Pull request merged**